fake_spectra/spectra.py

```diff
@@ -606,11 +612,21 @@ def replace_not_DLA(self, ndla, thresh=10**20.3, elem="H", ion=1):
         #Declare variables
         found = 0
         wanted = ndla
+        size_ind = None
         cofm_DLA = np.empty_like(self.cofm)[:ndla, :]
         #Filter
         #Note: line does nothing
         col_den = self.compute_spectra(elem,ion,1215,False)
-        ind = self.filter_DLA(col_den, thresh)
+        cdsum=np.sum(col_den, axis=1)
+        # A variable for comm.Reduce()
+        col_den_added = np.zeros_like(cdsum, dtype='d')
+        #ind = self.filter_DLA(col_den, thresh)
+        
+        ### Call manager rank here
+        self.comm.Reduce(cdsum, col_den_added, op=self.MPI.SUM, root=0)
+        size_ind = self.comm.bcast(size_ind, root=0)
+        ind = np.zeros(size_ind, dtype='d')
+        self.comm.Bcast(ind, root=0)
```

Comment:
You are broadcasting an array which is zero here. Is this placeholder code for future reimplementation of the filter_DLA function?